src/app/containers/ArticleMain/__snapshots__/index.test.jsx.snap

```diff
@@ -1,1360 +1,1045 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`should render a news article correctly 1`] = `
```

Comment:
Looks good to me. 
Just wondering why there's such a big change in this snapshot?